src/styles/page/page.scss

```
@@ -0,0 +1,30 @@
+@import "./functions";
+
+.card-holder {
+  width: minWidth(4);
+  padding: 50px 40px 74px;
+  display: flex;
+  margin: auto;
+  align-items: center;
+  flex-wrap: wrap;
+
+  @media (min-width: 1200px) {
+    & {
+      width: maxWidth(4);
+      padding-left: 128px;
+      padding-right: 128px;
+    }
+  }
```

```suggestion
  @media (min-width: 1200px) {
   padding-left: 128px;
   padding-right: 128px;
  }
```

Media query will be applied to its container automatically in scss.